packages/open-api-gateway/src/project/open-api-gateway-java-project.ts

```diff
@@ -0,0 +1,253 @@
+/*********************************************************************************************************************
+ Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+
+ Licensed under the Apache License, Version 2.0 (the "License").
+ You may not use this file except in compliance with the License.
+ You may obtain a copy of the License at
+
+ http://www.apache.org/licenses/LICENSE-2.0
+
+ Unless required by applicable law or agreed to in writing, software
+ distributed under the License is distributed on an "AS IS" BASIS,
+ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ See the License for the specific language governing permissions and
+ limitations under the License.
+ ******************************************************************************************************************** */
+
+import * as path from "path";
+import { DependencyType, Project, SampleDir } from "projen";
+import { JavaProject, JavaProjectOptions } from "projen/lib/java";
+import { ClientSettings } from "./codegen/components/client-settings";
+import { DocsProject } from "./codegen/docs-project";
+import { generateClientProjects } from "./codegen/generate";
+import { GeneratedJavaClientProject } from "./codegen/generated-java-client-project";
+import { ClientLanguage } from "./languages";
+import { getJavaSampleSource, JavaSampleCodeOptions } from "./samples/java";
+import { OpenApiSpecProject } from "./spec/open-api-spec-project";
+import { OpenApiGatewayProjectOptions } from "./types";
+
+const OPENAPI_GATEWAY_PDK_PACKAGE_NAME =
+  "software.aws.awsprototypingsdk/open-api-gateway@^0";
```

Comment:
well need to remember to bump this when we release 1.0

Reply:
Yeah, annoying!